docs/source/paper_index.md

```diff
@@ -457,6 +473,74 @@ training_args = DPOConfig(
 
 These parameters only appear in the [published version](https://aclanthology.org/2025.tacl-1.22.pdf)
 
+### Statistical Rejection Sampling Improves Preference Optimization
+**📜 Paper**: https://huggingface.co/papers/2309.06657
+
+Proposes **RSO**, selecting stronger preference pairs via statistical rejection sampling to boost offline preference optimization; complements DPO/SLiC.
+```python
+# Curate DPO pairs with rejection sampling BEFORE training
+from datasets import Dataset
+from trl import DPOConfig, DPOTrainer
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+def rso_accept(ex):  # replace with your statistic (gap / z-score / judge score)
+    return ex.get("rso_keep", True)
+
+dpo_pairs = dpo_pairs.filter(rso_accept)
+
+model = AutoModelForCausalLM.from_pretrained("..."); tok = AutoTokenizer.from_pretrained("...")
+args = DPOConfig(loss_type="sigmoid", beta=0.1)
+trainer = DPOTrainer(model=model, args=args, tokenizer=tok, train_dataset=dpo_pairs)
+trainer.train()
+```
+
+### Nash Learning from Human Feedback
+**📜 Paper**: https://huggingface.co/papers/2312.00886
+
+Frames alignment as a **two-player game**, learning Nash policies from human feedback; connects to multi-objective and competitive preference training.
+```python
+# Train a DPO policy (one side of the game-theoretic setup)
+from trl import DPOConfig, DPOTrainer
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+model = AutoModelForCausalLM.from_pretrained("..."); tok = AutoTokenizer.from_pretrained("...")
+args = DPOConfig(loss_type="sigmoid", beta=0.1)
+trainer = DPOTrainer(model=model, args=args, tokenizer=tok, train_dataset=...)
+trainer.train()
+
+```
+
+
+### Direct Language Model Alignment from Online AI Feedback
+**📜 Paper**: https://huggingface.co/papers/2402.04792
+
+Uses **online AI feedback (OAIF)** to supply real-time preference signals, improving direct alignment beyond purely offline pairs.
+```python
+# Sketch: collect online AI feedback -> extend DPO dataset -> train
+from trl import DPOConfig, DPOTrainer
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+policy = AutoModelForCausalLM.from_pretrained("..."); tok = AutoTokenizer.from_pretrained("...")
+
+def ai_feedback_score(prompt, response) -> float:
+    # plug a judge / RM / heuristic here (return scalar)
+    pass
+
+new_pairs = []
+for ex in prompts_ds:
+    out = policy.generate(**tok(ex["prompt"], return_tensors="pt").to(policy.device), max_new_tokens=256)
+    resp = tok.decode(out[0], skip_special_tokens=True)
+    score = ai_feedback_score(ex["prompt"], resp)
+    # build a (chosen, rejected) pair using score (e.g., compare vs baseline response)
+    new_pairs.append({"prompt": ex["prompt"], "chosen": resp, "rejected": ex["baseline"]})
+
+augmented_pairs = dpo_pairs.add_items(new_pairs)
+
+args = DPOConfig(loss_type="sigmoid", beta=0.1)
```

Comment:
there no need to pass the value if it matches the default. In other words, remove any occurence of like `loss_type="sigmoid"` or  `beta=0.1`